builtin/v13/market/invariants.go

```diff
@@ -157,10 +157,6 @@ func CheckStateInvariants(st *State, store adt.Store, balance abi.TokenAmount, c
 
 			dealStateCount++
 
-			if dealState.VerifiedClaim != verifreg.NoAllocationID {
```

Comment:
This is dropped, since deal states no longer record verified claims. New invariants will be added in a future PR.